lib/improver/wind_direction.py

```diff
@@ -163,34 +197,60 @@ def complex_to_deg(complex_in):
 
         return angle
 
-    @staticmethod
-    def find_r_values(complex_in):
+    def wind_dir_mean(self):
+        """Find the mean wind direction using complex average which actually
+           signifies a point between all of the data points in POLAR
+           coordinates - NOT the average DEGREE ANGLE.
+
+        Uses:
+            self.wdir_complex (np.ndarray or float):
+                3D array or float - wind direction angles in degrees.
+            self.realization_axis (int):
+                Axis to collapse over.
+
+        Defines:
+            self.wdir_mean_complex (np.ndarray or float):
+                3D array or float - wind direction angles as complex numbers
+                collapsed along an axis using np.mean().
+            self.wdir_slice_mean (np.ndarray or float):
+                3D array or float - wind direction angles in degrees collapsed
+                along an axis using np.mean().
+        """
+        self.wdir_mean_complex = np.mean(self.wdir_complex,
+                                         axis=self.realization_axis)
+        self.wdir_slice_mean.data = self.complex_to_deg(self.wdir_mean_complex)
+        return
```

Comment:
You've got a null "return" value here, with no "return" statement in some other functions.  Please choose one style or the other - most of IMPROVER I think goes with no return statement, so that's probably the better option.

Reply:
Agreed.